udata_front/theme/gouvfr/assets/js/composables/useDropdown.ts

```diff
@@ -2,7 +2,7 @@ import { ref, onMounted, onUnmounted, MaybeRefOrGetter, toValue } from 'vue';
 import { useCollapse } from './useCollapse';
 import useActiveDescendant from './useActiveDescendant';
 
-export function useDropdown(options: MaybeRefOrGetter, q: MaybeRefOrGetter<string>) {
+export function useDropdown(options: MaybeRefOrGetter, q: MaybeRefOrGetter<string>, linkKey: string) {
```

Comment:
If we want to remove `searchSelectedOption` from the components and move it here, I think we should introduce a type to ensure all `options` have the same key, instead of relying on a string parameter. Or at least we should use a [keyof type](https://www.typescriptlang.org/docs/handbook/2/keyof-types.html) to ensure that `linkKey` is actually in our options.